README.md

```diff
@@ -665,9 +665,58 @@ strict-transport-security:  max-age=15724800;
 includeSubDomains 
 ``` 
 
-**WebHook**
+#### WebHook
+##### Overview
+
+This system notifies the status changes of your transactions via a webhook. Please ensure you have configured a webhook URL in your Clip portal. For more information, refer to the [Clip Webhook Documentation](https://developer.clip.mx/reference/referencia-postback-webhook).
+
+##### Notification Events
+
+The application will notify you of the following transaction status changes:
+
+- `PENDING`
+- `IN_PROCESS`
+- `CANCELED`
+- `REJECTED`
+- `APPROVED`
+
+##### Webhook Notification Details
+
+When a transaction status changes, the application will send a POST request to the URL you have configured. The body of the request will be as follows:
+
+```json
+{
+	"id": "pinpad-51bf57ad-a186-4ac4-8a88-22133s8829",
+	"origin": "pinpad-payments-api",
+	"event_type": "PINPAD_INTENT_STATUS_CHANGED"
+}
+```
+
+##### Polling Strategy
+
+We use a polling strategy for notifications. This means that our system will notify you that there is an update for a specific transaction. To retrieve the details of the update, you need to make a GET request to the following URL:https://api.payclip.io/f2f/pinpad/v1/payment?requestId={id}
+
+
+Example: `GET https://api.payclip.io/f2f/pinpad/v1/payment?pinpadRequestId=pinpad-51bf57ad-a186-4ac4-8a88-22133s8829`
```

Comment:
Can we reformat this request like "Create Payment" section? 
https://github.com/ClipMX/mobile.android.blaze.pinpad.sdk/tree/docs/webhook-integration?tab=readme-ov-file#post-method

Reply:
Done